CONTRIBUTING.md

```diff
@@ -100,6 +100,21 @@ $ git push fork feature
 
 Open a pull request against the main `opentelemetry-python` repo.
 
+Pull requests are also tested for their compatibility with packages distributed 
+by OpenTelemetry in the [OpenTelemetry Python Contrib Repository](https://github.com/open-telemetry/opentelemetry-python.git).
+
+If a pull request introduces a change that would break the compatibility of
+these packages with the Core packages in this repo, a separate PR should be
+opened in the Contrib repo with changes to make the packages compatible.
+
+Once the pull request in the Contrib repo is open, take note of the commit SHA
+and edit the `CONTRIB_SHA` variable found in `.github/workflows/test.yml` to
+pass even the Contrib repo tests and so get the Core repo PR green and ready to
+merge.
```

Comment:
They will have to change it back to master before merging right? Would be nice to have some guidance for that

Reply:
I hadn't imagined them needing to change it back to `master`, but it's a good idea.

1. Make Core PR
2. Make Contrib PR
3. Point Core -> Contrib
4. Point Contrib -> Core
5. Merge Core
6. Point Contrib -> Core @ Master
7. Merge Contrib